app/build.gradle.kts

```diff
@@ -17,7 +17,6 @@ detekt {
     buildUponDefaultConfig = true
     allRules = false
     config = files("${rootProject.projectDir}/detekt.yml")
-    ignoreFailures = true
```

Comment:
If we add the linter workflow from the SDK this should be kept.. Otherwise the report won't be complete for the GitHub UI.